dashboard/db/migrate/20160630000000_add_pd_enrollment_survey_fields.rb

```diff
@@ -0,0 +1,9 @@
+class AddPdEnrollmentSurveyFields < ActiveRecord::Migration
+  def change
+    add_column :pd_enrollments, :survey_sent_at, :datetime
+    add_column :pd_enrollments, :survey_id, :integer
+
+    # This index should have been here all along
```

Comment:
Ok, I'l split it out. Any idea why it wasn't there in the first place, after `add_column :pd_enrollments, :code, :string, index: true, unique: true` in the original [migration](https://github.com/code-dot-org/code-dot-org/blob/af9eecff871ed981944c4d7ed9dcefabb0197af2/dashboard/db/migrate/20160510000000_add_district_school_and_code_to_enrollment.rb#L5-L5)? I would have thought that would create the index along with the column.